explorer/explorerroutes.go

```diff
@@ -142,26 +142,80 @@ func (exp *explorerUI) DisapprovedBlocks(w http.ResponseWriter, r *http.Request)
 
 // NextHome is the page handler for the "/nexthome" path.
 func (exp *explorerUI) NextHome(w http.ResponseWriter, r *http.Request) {
+	// get top 30 blocks and trim each block to have just the fields required for this page
```

Comment:
All comments in sentence form should begin with a capital letter and end with punctuation (.)